lisa/tools/lsblk.py

```diff
@@ -126,10 +126,10 @@ def can_install(self) -> bool:
         return True
 
     def _install(self) -> bool:
-        if isinstance(self.node.os, Posix):
-            self.node.os.install_packages("util-linux")
-        elif isinstance(self.node.os, FreeBSD):
+        if isinstance(self.node.os, FreeBSD):
             self.node.os.install_packages("lsblk")
+        elif isinstance(self.node.os, Posix):
+            self.node.os.install_packages("util-linux")
         else:
```

Comment:
oh, my bad, I didn't realize that BSD is from Posix, use Linux instead of Posix should also work.